index.html

```diff
@@ -0,0 +1,178 @@
+            <!-- <div class="speaker"> -->
+                <!-- <img src="mooc-images/speaker3.png" alt="the third speaker"> -->
+                <!-- <div class="him"> -->
+                    <!-- <h4>Lila Tretkov</h4> -->
+                    <!-- <p class="p"><em>Executive Director of the online program located in Eastern Europe</em></p> -->
+                    <!-- <hr> -->
+                    <!-- <p><em>Lila is the Executive Director of online foundation program, the non-profit organisation that works along wikipedia</em></p> -->
+                <!-- </div> -->
+            <!-- </div> -->
+            <!-- <div class="speaker"> -->
+                <!-- <img src="mooc-images/speaker4.png" alt="the fourth speaker"> -->
+                <!-- <div class="him"> -->
+                    <!-- <h4>Kilnam Chon</h4> -->
+                    <!-- <p class="p"><em>Chief Executive working in promotion of remote education via different online platform</em></p> -->
+                    <!-- <hr> -->
+                    <!-- <p><em>She volunteered in helping many online to apply diffferent approaches for the success of those platforms. </em></p> -->
+                <!-- </div> -->
+            <!-- </div> -->
+            <!-- <div class="speaker"> -->
+                <!-- <img src="mooc-images/speaker5.png" alt="the fifth speaker"> -->
+                <!-- <div class="him"> -->
+                    <!-- <h4>Julia Leda</h4> -->
+                    <!-- <p class="p"><em>Director and co-founder of the mooc-for-all school based in California</em>.</p> -->
+                    <!-- <hr> -->
+                    <!-- <p><em>She primarly started his early stage in computing in the 80's, then immigrated in the US in the 2000's where he encountered many experts in the domain.</em></p> -->
+                <!-- </div> -->
+            <!-- </div> -->
+            <!-- <div class="speaker"> -->
+                <!-- <img src="mooc-images/speaker6.png" alt="the sixth speaker"> -->
+                <!-- <div class="him"> -->
+                    <!-- <h4>Ryan Mekler</h4> -->
+                    <!-- <p class="p"><em>CEO and creative personnel chief at Tech promotion based in Ontario, Canada</em></p> -->
+                    <!-- <hr> -->
+                    <!-- <p><em>He started his career in civil engineering and later moved to the tech industry where he worked in many organizations and companies based precisely in North-America.</em></p> -->
+                <!-- </div> -->
+            <!-- </div> -->
+        </section>
+        <button type="button" class="more">MORE<img src="mooc-images/arrow.png" alt="the arrow to find more"></button>
+        <div class="partners">
+            <h3>Partner</h3>
+            <hr>
+            <ul>
+                <li>Microverse</li>
+                <li>Odin</li>
+                <li>HackerRank</li>
+                <li>freeCode</li>
+                <li>codeCademy</li>
+            </ul>    
+        </div>
```

Comment:
- [ ] For the `Partners` section, please again make sure you stick to the design by using the given colors and fonts in the original design. Below you can see the design differences between yours and the given one, please fix them 👇 
- Original 👇 
![Screenshot (189)](https://user-images.githubusercontent.com/67087664/153600397-e6e43a5b-03c0-4eba-ab35-a013e97b1d4a.png)

- Yours 👇 
![Screenshot (188)](https://user-images.githubusercontent.com/67087664/153600375-0f35c7ce-5084-4561-a562-62f3a654ec21.png)